fairlearn/widget/FairlearnDashboard.py

```diff
@@ -0,0 +1,122 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+"""Defines the Fairlearn dashboard class."""
+
+from .FairlearnWidget import FairlearnWidget
+from IPython.display import display
```

Comment:
are there any new dependencies we need to add now that we don't have a separate package anymore? I mean dependencies that we'd need to add to setup.py & requirements.txt

Reply:
the setup.py has been updated to include all the widget pieces in the python wheel (I hope)